README.markdown

```diff
@@ -10,220 +10,134 @@ There are two ways in which you can install OneZoom on a personal computer: full
-;    defined, will default to the local version, but that
-;    means you will need to download >100,000 thumbnail images
-;    onto your machine. If you want to use the images on the
-;    OneZoom server, set this to `//images.onezoom.org/`
-url_base = //images.onezoom.org/
-
-[sponsorship]
-; * allow. Should we allow the sponsorship page to be
-;    shown on this machine? Usually not allowed, except on the
-;    main OneZoom site (on museum displays people will not want
-;    to enter paypal etc details).
-; * maintenance_mins: to enable maintenance mode (e.g. when 
-;    switching beta and production to enable a new website version)
-;    set to the number of minutes you expect the site to be down
-;    note that if is_testing=False then you will probably need to 
-;    restart the server for changes to this to take effect
-; * reservation_time_limit_mins: how long to reserve a leaf while a 
-;    user is looking at the sponsor page
-; * unpaid_time_limit_mins: how long before a sponsored leaf becomes 
-;    free again if we receive no payment notification
-allow_sponsorship = 0
-maintenance_mins = 0
-
-[api]
-;If you want to get data from the Encyclopedia of Life, you need to put your own API key here. 
-;Fill it in using instructions at http://eol.org/info/api_overview 
-;eol_api_key = 11111111111
+# NB: OneZoom.dump.sql can be extracted from https://github.com/OneZoom/OZtree-docker
+mysql -p
+USE OneZoom
+SOURCE /OneZoom.dump.sql
 ```
 
-In order to use web2py you need to have a python v3 installed, the latest version can be found at
-[https://www.python.org/downloads/](https://www.python.org/downloads/)
+Finally, ensure indexes are created:
 
-NB: on Windows, make sure that you add `python` (and ideally `python3`) to the windows path during install, or the commands below will not work
-
-Assuming you have python version 3 installed, should now try starting web2py as follows.
+```
+mysql -p
+USE OneZoom
+SOURCE ${WEB2PY_PATH}/applications/OZtree/OZprivate/ServerScripts/SQL/create_db_indexes.sql
+```
 
-### Starting and shutting down web2py
+## Database explorer
 
-On the OneZoom main site, web2py is run using a combination of nginx and uwsgi. This is complete overkill if you just want to run a local copy of OneZoom for testing purposes. You can simply run a [temporary and basic web2py server using Python 3](http://www.web2py.com/books/default/chapter/29/03/overview#Startup). The simplest is to open a command-line prompt in the root web2py folder, and run the following (assuming the command `python3` is linked to something like Python 3.7)
+(optional) We find it useful to have a GUI interface to connect to the database and run SQL scripts, this can be used instead of using MySQL command line (similar to Windows command line) that is installed by default with MySQL. On Mac OS X we use the (excellent) http://www.sequelpro.com. On windows you could try http://www.mysql.com/products/workbench/ or https://www.quest.com/products/toad-for-mysql/
```

Comment:
Worth noting that when using WSL2, installing SQL Workbench on Windows works great to connect to the Ubuntu MySQL instance.